aries_cloudagent/indy/models/cred.py

```diff
@@ -155,3 +155,42 @@ class Meta:
     witness = fields.Dict(
         allow_none=True, metadata={"description": "Witness for revocation proof"}
     )
+
```

Comment:
I'd prefer if these changes are moved under the `formats/vc_di` folder (maybe a `models` sub-folder?) to keep the dependencies cleaner.  vc_di is the exchange format (vs the anoncreds/indy credential format) so if all the vc_di related stuff is under the `formats/vc_di` it will be cleaner.